### PR TITLE
updating mentions of overseas on the website

### PR DESCRIPTION
**Pull request by Sarah-DfE**
### Trello card
https://trello.com/c/agrQMMx1/6878-replace-overseas-on-all-pages?filter=member:sarahstewart179#

### Context
We currently use the word ‘overseas’ in some of the international content.

However, we know from user research carried out by the international team that international candidates wouldn’t think of themselves as ‘overseas’.

We should therefore replace the word overseas (eg ‘outside the UK’) where appropriate.

### CODE REVIEW

File: app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ related_content:
 promo_content:
 navigation: 20.40
 navigation_title: Teach in England as a non-UK qualified teacher
-navigation_description: Find out how you can bring your skills and perspective to an English classroom if you're a qualified teacher from overseas.
+navigation_description: Find out how you can bring your skills and perspective to an English classroom if you qualified to teach outside the UK.
```
Comment (emmaachesongray) [inline app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md:13]: Should this be 'perspectives'?

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md [hunk 2]
```diff
@@ -517,8 +517,8 @@ Talk to your provider about whether your experience makes you eligible for this
 
 Or, find other ways you can [train to be a teacher](/train-to-be-a-teacher).
 
-### If you're based overseas or are a non-UK citizen
+### If you're based outside the UK or are a non-UK citizen
```
Comment (emmaachesongray) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:520]: Might be clearer to say 'live outside the UK'? how do we define 'based'?

File: app/views/teacher_training_adviser/steps/_uk_callback.html.erb [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 <% if f.object.class.callback_booking_quotas.any? %>
   <p>You need to book a callback with us to check your degree. We will contact you once your call is booked.</p>
-  <p>You must have the details of your overseas qualifications when we contact you.</p>
+  <p>You must have the details of the qualification you gained outside the UK when we contact you.</p>
```
Comment (emmaachesongray) [inline app/views/teacher_training_adviser/steps/_uk_callback.html.erb:7]:
```suggestion
  <p>You must have the details of the qualifications you gained outside the UK when we contact you.</p>
```

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md (hunk 2 above)
Comment (Sarah-DfE) [reply app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:520]:
```suggestion
### If you live outside the UK or are a non-UK citizen
```

File: config/locales/en.yml [hunk 4]
```diff
@@ -702,7 +702,7 @@ en:
         address_telephone: "Contact telephone number"
         time_zone: "Select your time zone"
       teacher_training_adviser_steps_overseas_telephone:
-        address_telephone: "Overseas telephone number (optional)"
+        address_telephone: "Non-UK telephone number (optional)"
```
Comment (Sarah-DfE) [inline config/locales/en.yml:705]:
```suggestion
        address_telephone: "Overseas telephone number (optional)"
```

File: app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb [hunk 5]
```diff
@@ -1,6 +1,6 @@
-<% @page_title = "You told us you have an equivalent degree and live overseas" %>
+<% @page_title = "You told us you have an equivalent degree and live outside the UK" %>
```
Comment (Sarah-DfE) [inline app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb:1]:
```suggestion
<% @page_title = "You told us you have an equivalent degree and live overseas" %>
```

File: app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb [hunk 6]
```diff
@@ -1,6 +1,6 @@
...
 
-<%= f.govuk_fieldset legend: { text: "You told us you have an equivalent degree and live overseas", tag: "h1" } do %>
+<%= f.govuk_fieldset legend: { text: "You told us you have an equivalent degree and live outside the UK", tag: "h1" } do %>
```
Comment (Sarah-DfE) [inline app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb:3]:
```suggestion
<%= f.govuk_fieldset legend: { text: "You told us you have an equivalent degree and live overseas", tag: "h1" } do %>
```

File: app/views/teacher_training_adviser/steps/_uk_callback.html.erb [hunk 7]
```diff
@@ -14,7 +14,7 @@
 <% else %>
   <p>We need to check your degree.</p>
   <p>Call us on <%= link_to("0800 389 2500", "tel://08003892500") %> between 8.30am and 5.30pm (UK time).</p>
-  <p>Please have the details of your overseas qualifications to hand when you call.</p>
+  <p>Please have the details of the qualifications you gained outside the UK to hand when you call.</p>
```
Comment (Sarah-DfE) [inline app/views/teacher_training_adviser/steps/_uk_callback.html.erb:17]:
```suggestion
  <p>Please have the details of your overseas qualifications you gained outside the UK to hand when you call.</p>
```

File: app/views/teacher_training_adviser/steps/_uk_callback.html.erb [hunk 8]
```diff
@@ -14,7 +14,7 @@
 <% else %>
   <p>We need to check your degree.</p>
   <p>Call us on <%= link_to("0800 389 2500", "tel://08003892500") %> between 8.30am and 5.30pm (UK time).</p>
-  <p>Please have the details of your overseas qualifications to hand when you call.</p>
+  <p>Please have the details of your overseas qualifications you gained outside the UK to hand when you call.</p>
```
Comment (Sarah-DfE) [inline app/views/teacher_training_adviser/steps/_uk_callback.html.erb:17]:
```suggestion
  <p>Please have the details of your overseas qualifications to hand when you call.</p>
```

**Review by gemmadallmandfe — COMMENTED** [review]
This looks good to me @Sarah-DfE 

There is a merge conflict that needs sorting out (I will tick it after that) and then it should probably have a tech review given the funnel changes

Let's get this out today :-)

File: app/views/content/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers.md [hunk 9]
```diff
@@ -64,14 +64,16 @@ If you're interested in exploring other opportunities to work with pre-school ch
...
+To do postgraduate training, you’ll need a bachelor’s degree or an equivalent, for example, a vocational or professional qualification, or equivalent qualification you gained outside the UK.
 
-To start early years teacher training, you’ll need these GCSE qualifications (or a vocational, professional, or overseas qualification to the same standard):
+To start early years teacher training, you’ll need these GCSE qualifications:
 
 * maths grade 4 (C)
 * English grade 4 (C)
 * science grade 4 (C)
 
+If you do not have GCSEs, you'll need an equivalent qualifiaction, such as a vocational or professional qualification, or an equivalent qualification you gained outside the UK.
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers.md:75]:
```suggestion
If you do not have GCSEs, you'll need an equivalent qualification, such as a vocational or professional qualification, or an equivalent qualification you gained outside the UK.
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me from a front-end perspective

**Review by martyn-w — APPROVED** [review]
👍 LGTM

**Pull request merged**